integration/nwo/network.go

```diff
@@ -770,18 +744,13 @@ func (n *Network) GenerateConfigTree() {
 	}
 }
 
-// Bootstrap generates the cryptographic material, orderer system channel
-// genesis block, and create channel transactions needed to run a fabric
-// network.
+// Bootstrap generates the cryptographic material and create channel transactions
+// needed to run a fabric network.
 //
 // The cryptogen tool is used to create crypto material from the contents of
 // ${rootDir}/crypto-config.yaml. The generated artifacts will be placed in
 // ${rootDir}/crypto/...
 //
-// The gensis block is generated from the profile referenced by the
-// SystemChannel.Profile attribute. The block is written to
-// ${rootDir}/${SystemChannel.Name}_block.pb.
-//
```

Comment:
Keep this comment but update it:

The gensis block is generated from the profile referenced by the channel's Profile attribute. The block is written to ${rootDir}/${Channel.Name}_block.pb.

(I hope I got the path right, check it).

Reply:
You are correct:
`filepath.Join(n.RootDir, fmt.Sprintf("%s_block.pb", channelName))`.